tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -373,3 +375,46 @@ def test_match_in_holdingpen_different_sources_continues(
     assert obj.extra_data['holdingpen_matches'] == [wf_to_match]
     assert obj.extra_data['previously_rejected'] is False
     assert not obj.extra_data.get('stopped-matched-holdingpen-wf')
+
+
+@mock.patch(
+    'inspirehep.modules.workflows.tasks.arxiv.is_pdf_link',
+    return_value=True
+)
+@mock.patch(
+    'inspirehep.modules.workflows.tasks.arxiv.download_file_to_workflow',
+    side_effect=fake_download_file,
+)
+@mock.patch(
+    'inspirehep.modules.workflows.tasks.beard.json_api_request',
+    side_effect=fake_beard_api_request,
+)
+@mock.patch(
+    'inspirehep.modules.workflows.tasks.magpie.json_api_request',
+    side_effect=fake_magpie_api_request,
+)
+def test_arxiv_update_overwrites_record_on_labs_but_not_in_legacy(
```

Comment:
why should it overwrite the record on labs? don't we want to maintain it in sync with legacy?

Reply:
For now yes, and in fact, after the robot upload the record comes back from legacy and overwrites whatever there is.

Reply:
but if it's an update, there is no robotupload, so the record might never come back.

Reply:
That's true, you are right. So, let's throw them away.